### PR TITLE
Docs on searchpath modification for unittesting

### PR DESCRIPTION
**Pull request by Jasha10**
This PR updates the docs with a few hints about how to modify Hydra's searchpath when performing unit testing.

This is a follow-up to #2001.

### CODE REVIEW

File: website/versioned_docs/version-1.1/advanced/compose_api.md [hunk 1]
```diff
@@ -111,3 +111,6 @@ def initialize_config_dir(config_dir: str, job_name: str = "app") -> None:
     """
 ```
 
+For an idea about how to modify Hydra's search path when using `compose` in
+unit tests, see the page on
+[overriding the `hydra.searchpath` config](search_path.md#overriding-hydrasearchpath-config).
```
Comment (jieru-hu) [inline website/versioned_docs/version-1.1/advanced/compose_api.md:116]: Thank you! Why specifically mention unit tests? This feature is more for a general purpose - making configs available in other dirs.

**Review by jieru-hu — CHANGES_REQUESTED** [review]
see comment

File: website/versioned_docs/version-1.1/advanced/compose_api.md (hunk 1 above)
Comment (Jasha10) [reply website/versioned_docs/version-1.1/advanced/compose_api.md:116]: Maybe the compose doc isn't the right place for the searchpath hint; searchpath modification is not just relevant for `compose`, but also for `@hydra.main` (which is the alternative to compose).

How about I move this unit test hint to the page on [unit testing](https://hydra.cc/docs/advanced/unit_testing/)?

Comment (Jasha10) [reply website/versioned_docs/version-1.1/advanced/compose_api.md]: See the latest commit (https://github.com/facebookresearch/hydra/pull/2007/commits/130bde1bcf1595c608209de8fe1fede16a7591e9).

**Review by jieru-hu — APPROVED** [review]
lg! thank you! 🚀

**Pull request merged**